sympy/liealgebras/type_a.py

```diff
@@ -69,9 +68,10 @@ def simple_root(self, i):
         [1, -1, 0, 0, 0]
 
         """
-
+        if i < 1 or i > self.n: raise ValueError("Simple root does not exist.")
```

Comment:
The raise statement could be on another line and the message could be something like: "Simple root %s of type A_%s does not exist." % (i, n).

Reply:
Will change in next commit

Reply:
will fix in next commit